content/sources/repositories.adoc

```diff
@@ -1,8 +1,40 @@
 = Git - Repositories
 
-== Platform codebase
+== Platform 6
 
-=== Platform 5
+eXo Platform is built from several projects stored in different {link-github-exoplatform}[*Git repositories*].
+You will find below the list of repositories used to build *eXo Platform 6*.
+
+.Overview of eXo Components for PLF 6.0
+image::plf6-components.png[Platform Components, align=center, link={imagesdir}/plf6-components.png]
```

Comment:
The addons digital-workplace, layout-management and news are missing in the document.

Reply:
done